low-level/framework/base/sspl_constants.py

```diff
@@ -57,18 +57,73 @@
 CONSUL_HOST = consulhost
 CONSUL_PORT = consulport
 SSPL_SETTINGS = {
-        "ACTUATORS" : ["Service", "RAIDactuator", "Smartctl", "NodeHWactuator"],
-        "CORE_PROCESSORS" : ("RabbitMQegressProcessor", "RabbitMQingressProcessor", "LoggingProcessor"),
-        "DEGRADED_STATE_MODULES" : ("ServiceWatchdog", "RAIDsensor", "NodeData", "IEMSensor", "NodeHWsensor",
-                            "DiskMsgHandler", "LoggingMsgHandler", "ServiceMsgHandler", "NodeDataMsgHandler",
-                            "NodeControllerMsgHandler", "SASPortSensor", "MemFaultSensor", "CPUFaultSensor"),
-        "MESSAGE_HANDLERS" : ("DiskMsgHandler", "LoggingMsgHandler", "ServiceMsgHandler", "NodeDataMsgHandler",
-                        "NodeControllerMsgHandler"),
-        "SENSORS" : ["ServiceWatchdog", "RAIDsensor", "NodeData",
-            "IEMSensor", "NodeHWsensor",
-            "SASPortSensor", "MemFaultSensor", "CPUFaultSensor", "RAIDIntegritySensor"]
+        "REALSTORSENSORS": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["RealStorFanSensor", "RealStorPSUSensor",
+                "RealStorControllerSensor", "RealStorDiskSensor", "RealStorSideplaneExpanderSensor",
+                "RealStorLogicalVolumeSensor", "RealStorEnclosureSensor"],
+        },
+        "NODEHWSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [ "NodeHWsensor"],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": [ "NodeHWsensor"],
+        },
+        "SYSTEMDWATCHDOG": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": [],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": [],
+        },
+        "RAIDSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": ["RAIDsensor"],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["RAIDsensor", "RAIDIntegritySensor"],
+        },
+        "SASPORTSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": ["SASPortSensor"],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["SASPortSensor"],
+        },
+        "MEMFAULTSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": ["MemFaultSensor"],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["MemFaultSensor"],
+        },
+        "CPUFAULTSENSOR": {
+            "ACTUATORS": [],
+            "CORE_PROCESSORS": [],
+            "DEGRADED_STATE_MODULES": ["CPUFaultSensor"],
+            "MESSAGE_HANDLERS": [],
+            "SENSORS": ["CPUFaultSensor"],
+        },
+
+        "CATCHALL": {
```

Comment:
CATCHALL means this set will be enabled for all config types right ? ENABLE_ALWAYS name would suit ?

Reply:
I have changed the name to '_ENABLE_ALWAYS' with the leading underscore to further indicate that it is not a valid section name. I hope that's fine.

Reply:
yes